### PR TITLE
Add missing properties to the image

### PR DESCRIPTION
**Pull request by rumpl**
Including buildkit's custom properties

### CODE REVIEW

File: daemon/containerd/image.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/docker/docker/image"
 	"github.com/docker/docker/layer"
 	"github.com/docker/go-connections/nat"
+	buildkitimage "github.com/moby/buildkit/frontend/dockerfile/dockerfile2llb"
```
Comment (jedevc) [inline daemon/containerd/image.go:23]:
```suggestion
	"github.com/moby/buildkit/frontend/dockerfile/dockerfile2llb"
```

Comment (rumpl) [reply daemon/containerd/image.go:23]: Fair enough, fixed, thanks

**Review by vvoland — APPROVED** [review]

**Review by jedevc — APPROVED** [review]

File: daemon/containerd/image.go [hunk 2]
```diff
@@ -86,7 +87,7 @@ func (i *ImageService) getImage(ctx context.Context, refOrID string, platform *o
 		return nil, nil, err
 	}
 
-	var ociimage ocispec.Image
+	var ociimage dockerfile2llb.Image
```
[the remark this answers is not in the document]
Comment (rumpl) [reply daemon/containerd/image.go:90]: The last type you linked is an api type, we shouldn't use it for reading things, only for writing to it and sending to the caller. 

That being said then moby doesn't have any type that defines a HealthConfig.

[the remark this answers is not in the document]
Comment (thaJeztah) [reply daemon/containerd/image.go:90]: But how is the dockerfile2llb type different? That's also an API type; it's the type to export to JSON

[the remark this answers is not in the document]
Comment (rumpl) [reply daemon/containerd/image.go:90]: The dockerfile2llb type is used to store the image, that's why it's later moved over to the exporter. We should use the same type that is used to save the image when we load it. Then we can convert it to our own types.

[This type](https://github.com/moby/moby/blob/v23.0.0-beta.1/api/types/container/config.go#L35-L54) is the type that we use for the engine API. It should be independent of anything else.

[the remark this answers is not in the document]
Comment (rumpl) [reply daemon/containerd/image.go:90]: Put in another way: the dockerfile2llb type is different in a sense that it's a buildkit's type. We have our own for the engine API. They both look the same, but they are not the same

**Pull request merged**